robot-name/Pratster95/robot_name.go

```diff
@@ -0,0 +1,40 @@
+package robotname
+
+import (
+	"fmt"
+	"math/rand"
+)
+
+// Define the Robot type here.
+type Robot struct {
+	name string
+}
+
+var nameUsed = map[string]bool{}
```

Comment:
Correct me if I'm wrong, but the proper (most common) way of creating an empty map is:
```go
make(map[string]bool)
```
You don't need a literal declaration if you are not adding any key/value.